examples/camera/scene.js

```diff
@@ -24,13 +24,15 @@ export async function sceneBuilder(animationLoop) {
       easings: [easing.easeInOut]
     })
   };
-  animationLoop.timeline.attachAnimation(keyframes.camera);
+
+  const currentCamera = animationLoop.timeline.attachAnimation(keyframes.camera);
 
   return new DeckScene({
     animationLoop,
     keyframes,
     lengthMs: 5000,
     width: 640,
```

Comment:
Not sure if you're on this task yet but shouldn't this be modular? So that the user can set resolution themselves

Reply:
I don't think we've ever scoped or discussed setting resolution differently. At the moment it's designed to be set within scene definition since it's how I preferred to set up scenes. 

We could change this to be optional if DeckAdapter is changed to only set it if it's defined. Happy to accept a PR for that.

Reply:
Got it. I imagined those being values being set with state but I must have misremembered.

Reply:
Got it. I imagined those being values being set with state but I must have misremembered.